plugins/automox/plugin.spec.yaml

```diff
@@ -0,0 +1,988 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: automox
+title: Automox
+description: Automox is Modernizing IT operations with continuous visibility, insight, and agility for your entire IT environment
+version: 1.0.0
+supported_versions: ["All"]
+vendor: automox
+support: partner
+status: []
+tags: [endpoint, patch]
+hub_tags:
+  use_cases: [threat_detection_and_response, asset_inventory, remediation_management]
+  keywords: [endpoint]
+  features: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/automox
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.automox.com/
+enable_cache: false
+
+types:
+  organization:
+    id:
+      title: Organization ID
+      description: The organization identifier
+      type: integer
+      required: true
+    name:
+      title: Organization Name
+      description: The organization name
+      type: string
+      required: true
+    create_time:
+      title: Creation Time
+      description: The datetime of when the organization was created
+      type: string
+      required: false
+    server_limit:
+      title: Server Limit
+      description: The organization server limit
+      type: integer
+      required: false
+    parent_id:
+      title: Parent Organization ID
+      description: The parent organization identifier
+      type: integer
+      required: false
+    device_limit:
+      title: Device Limit
+      description: The organization device limit
+      type: integer
+      required: false
+    device_count:
+      title: Device Count
+      description: The organization device count
+      type: integer
+      required: false
+  users:
+    id:
+      title: User ID
+      description: The user identifier
+      type: integer
+      required: true
+    firstname:
+      title: First Name
+      description: The first name of the user
+      type: string
+      required: false
+    lastname:
+      title: Last Name
+      description: The last name of the user
+      type: string
+      required: false
+    email:
+      title: Email
+      description: The email of the user
+      type: string
+      required: false
+    features:
+      title: Features
+      description: The features enabled for the user
+      type: "object"
+      required: false
+    orgs:
+      title: Organizations
+      description: The organizations for which the user has access
+      type: "[]object"
```

Comment:
Could this be a []organization? If not, thats fine! Just seeing if we can reuse from above

Reply:
Unfortunately its different in this context and information can vary